docs/book/src/developer/providers/contracts/clusterctl.md

```diff
@@ -201,6 +201,25 @@ releaseSeries:
   contract: v1alpha2
 ```
 
+#### Validation Rules
+
+Starting from clusterctl v1.11, the metadata YAML file is subject to strict validation to ensure consistency and prevent configuration errors. The following validation rules are enforced:
+
+1. **apiVersion**: Must be set to `clusterctl.cluster.x-k8s.io/v1alpha3`
+   * This ensures compatibility with the current clusterctl metadata format
+   * Invalid apiVersion will result in an error: `invalid provider metadata: unexpected apiVersion "<value>" for provider <provider-name> (expected "clusterctl.cluster.x-k8s.io/v1alpha3")`
```

Comment:
I would not document the exact error messages here.

1. There's a very low chance we'll keep these up-to-date when the code changes
2. The exact wording of the error message should not be relevant for users
3. The error message format is not part of our API / contract

Reply:
Thx. I fixed.